### PR TITLE
feat: promo, review, services, baner sections

### PR DESCRIPTION
**Pull request by YaroslavBoldyshev**
promo, review, services, baner sections

### CODE REVIEW

File: components/layout/Image/index.tsx
Comment (okuruchimas) [inline components/layout/Image/index.tsx:1]: directory name need small letter

File: components/sections/company/promo/index.tsx [hunk 1]
```diff
@@ -1,7 +1,142 @@
...
+import type { FC } from "react";
+import type { PromCardI } from "../../../../pages/api/prom-cards";
 
-const Promo = () => {
-  return <div>promo</div>;
-};
+interface PromoProps
+  extends Pick<
+    PromCardI,
+    "images" | "discount" | "slug" | "title" | "location"
```
Comment (okuruchimas) [inline components/sections/company/promo/index.tsx:14]: what is it (props)

Comment (YaroslavBoldyshev) [reply components/sections/company/promo/index.tsx:14]: types for Promo component props

File: components/sections/company/review/index.tsx [hunk 2]
```diff
@@ -1,12 +1,114 @@
...
+interface ReviewFormValues {
+  email: string;
+  review: string;
+}
+
+const ReviewForm: FC = () => {
+  const [stars, setStars] = useState(0);
+
+  const handleSubmit = useCallback(
+    async (
```
[the remark this answers is not in the document]
Comment (YaroslavBoldyshev) [reply components/sections/company/review/index.tsx:24]: image

just checked it, i think yes.

**Pull request merged**